src/main/resources/static/index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sweet Rounds Donut Shop - Coming Soon!</title>
+    <!-- Bootstrap 5.3 CSS CDN -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.0/font/bootstrap-icons.css">
+    <style>
+        body {
+            background-color: #FFF8E1;
+            font-family: 'Roboto', sans-serif;
+        }
+        .hero-section {
+            background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('https://images.unsplash.com/photo-1556913396-7a3c459ef68e?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1470&q=80');
+            background-size: cover;
+            background-position: center;
+            color: white;
+            padding: 100px 0;
+        }
+        .countdown {
+            background-color: #FF85A2;
+            color: white;
+            padding: 20px;
+            border-radius: 10px;
+        }
+        .form-section {
+            background-color: white;
+            border-radius: 10px;
+            box-shadow: 0 4px 8px rgba(0,0,0,0.1);
+            padding: 30px;
+            margin-top: -50px;
+        }
+        .btn-primary {
+            background-color: #FF85A2;
+            border-color: #FF85A2;
+        }
+        .btn-primary:hover {
+            background-color: #FF6B8B;
+            border-color: #FF6B8B;
+        }
+        .feature-icon {
+            font-size: 2.5rem;
+            color: #FF85A2;
+            margin-bottom: 15px;
+        }
+    </style>
+</head>
+<body>
+    <!-- Hero Section -->
+    <section class="hero-section text-center">
+        <div class="container">
+            <h1 class="display-2 fw-bold mb-4">Sweet Rounds Donut Shop</h1>
+            <h2 class="display-6 mb-5">Freshly Baked Happiness Coming Soon!</h2>
+            <div class="row justify-content-center">
+                <div class="col-md-6">
+                    <div class="countdown mb-4">
+                        <h3>Grand Opening in</h3>
+                        <h2 class="display-4 fw-bold">30 Days</h2>
```

Comment:
The grand opening has already happened. Change this to come on down or something like that.